conda/xscen/meta.yaml

```diff
@@ -18,34 +18,37 @@ build:
 
 requirements:
   host:
-    - python >=3.8
+    - python >=3.9
     - pip
   run:
-    - python >=3.8
     - cartopy
     - cftime
     - cf_xarray >=0.7.6
-    - clisops >=0.9.5
-    - dask-core
+    - clisops >=0.10
+    - dask
```

Comment:
I can't remember, what are the differences between `dask` and `dask-core`? Does `dask` vendor `distributed` and other helpers?

Reply:
I think so yeah. Installing `dask` with conda will pull ``distributed`. I did not realize this difference as I only copy-pasted the env list... I can change it back to `-core`. `distributed` is indeed not necessary for xscen.